### PR TITLE
More intruments

### PR DESCRIPTION
**Pull request by xxAROX**
[Source](https://github.com/TuranicTeam/Altay/blob/stable/src/pocketmine/level/sound/NoteBlockSound.php)

### CODE REVIEW

**Comment by dktapps** [comment]
I don't consider Altay a reliable source. Please verify these against the wiki or some other source.

File: src/world/sound/NoteInstrument.php [hunk 1]
```diff
@@ -48,7 +59,18 @@ protected static function setup() : void{
...
-			new self("double_bass", 4)
+			new self("double_bass", 4),
+			new self("glockenspiel", 5),
+			new self("flute", 6),
+			new self("chime", 7),
+			new self("guitar", 8),
+			new self("xylophone", 9),
+			new self("vibraphone", 10),
+			new self("cow_bell", 11),
+			new self("diggeridoo", 12),
```
Comment (dktapps) [inline src/world/sound/NoteInstrument.php:70]: This is spelled incorrectly.

**Comment by xDidntPot** [comment]
wtf 🤣

**Comment by xxAROX** [comment]
found nothing in web

**Pull request closed without merging**